### PR TITLE
RavenDB-16311 OLAP ETL : Allow to test script

### PR DESCRIPTION
**Pull request by arekpalinski**
https://issues.hibernatingrhinos.com/issue/RavenDB-16311

### CODE REVIEW

File: test/SlowTests/Server/Documents/ETL/Olap/RavenDB_16311.cs [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+                                        Script =
+                                            @"
+                                                var orderDate = new Date(this.OrderedAt);
+                                                var year = orderDate.getFullYear();
+                                                var month = orderDate.getMonth();
+                                                var key = new Date(year, month);
+
+                                                output('test output')
+
+                                                loadToOrders(partitionBy([['order_date', key]]),
```
Comment (aviv86) [inline test/SlowTests/Server/Documents/ETL/Olap/RavenDB_16311.cs]: syntax was changed, need to remove the outer array i.e. `partitionBy(['order_date', key])`

File: test/SlowTests/Server/Documents/ETL/Olap/RavenDB_16311.cs [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+                                }
+                        }
+                    }, database, database.ServerStore, context);
+
+                    Assert.Equal(0, result.TransformationErrors.Count);
+
+                    Assert.Equal(1, result.ItemsByPartition.Count);
+
+                    Assert.Equal(4, result.ItemsByPartition[0].Columns.Count);
+
```
Comment (aviv86) [inline test/SlowTests/Server/Documents/ETL/Olap/RavenDB_16311.cs:104]: maybe also assert that `ItemsByPartition[0].Key ` = `Orders/order_date=2020-01-01` ?

File: test/SlowTests/Server/Documents/ETL/Olap/RavenDB_16311.cs (hunk 1 above)
Comment (arekpalinski) [reply test/SlowTests/Server/Documents/ETL/Olap/RavenDB_16311.cs]: done

File: test/SlowTests/Server/Documents/ETL/Olap/RavenDB_16311.cs (hunk 2 above)
Comment (arekpalinski) [reply test/SlowTests/Server/Documents/ETL/Olap/RavenDB_16311.cs:104]: Done. Although it is "Orders/order_date=2020-01-01-00-00"

**Pull request merged**